src/engine/bank20.asm

```diff
@@ -1434,16 +1455,16 @@ Tilesets: ; 8100f (20:500f)
 	tileset ScytherIntro2Gfx,               96 ; TILESET_SCYTHER_INTRO_2
 	tileset AerodactylIntro1Gfx,            96 ; TILESET_AERODACTYL_INTRO_1
 	tileset AerodactylIntro2Gfx,            96 ; TILESET_AERODACTYL_INTRO_2
-	tileset Titlescreen1Gfx,                97 ; TILESET_TITLE_SCREEN_1
-	tileset Titlescreen2Gfx,                97 ; TILESET_TITLE_SCREEN_2
+	tileset Titlescreen1Gfx,                97 ; TILESET_JAPANESE_TITLE_SCREEN
```

Comment:
While we're moving away from calling them `Titlescreen[1-6]`, might as well rename the Gfx labels and the pngs too (and I suppose capitalize the 'S' in 'Titlescreen' for consistency).